examples/visual-tests/amp-mega-menu/amp-mega-menu.js

```diff
@@ -19,6 +19,7 @@ const {verifySelectorsVisible} = require('../../../build-system/tasks/visual-dif
 
 module.exports = {
   'click menu item one': async (page, name) => {
+    await page.waitFor('amp-mega-menu .i-amphtml-mega-menu-item:first-child .i-amphtml-mega-menu-heading');
```

Comment:
Not required for now, but can we reduce the specificity of these? I don't think we need to specify `amp-mega-menu` as a parent if we're checking an internal classname.

Reply:
Will add an issue for this